src/Core/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -73,7 +73,7 @@ public class MsSqlQueryExecutor : QueryExecutor<SqlConnection>
                   httpContextAccessor)
         {
             RuntimeConfig runtimeConfig = runtimeConfigProvider.GetConfig();
-            IEnumerable<KeyValuePair<string, DataSource>> mssqldbs = runtimeConfig.GetDataSourceNamesToDataSourcesIterator().Where(x => x.Value.DatabaseType == DatabaseType.MSSQL);
+            IEnumerable<KeyValuePair<string, DataSource>> mssqldbs = runtimeConfig.GetDataSourceNamesToDataSourcesIterator().Where(x => x.Value.DatabaseType is DatabaseType.MSSQL || x.Value.DatabaseType is DatabaseType.DWSQL);
```

Comment:
Does the RuntimeConfig property about `SqlSourceUsed` need to be modified for multiple config scenario where one of the configs is Dw?

Reply:
I have modified the runtimeconfig sqlSourceUsed in this pr:
<img width="845" alt="image" src="https://github.com/Azure/data-api-builder/assets/124841904/b5915fec-9494-4b89-bd42-89bc7703b3b0">